src/api/config/encoder.rs

```diff
@@ -29,6 +29,8 @@ pub struct EncoderConfig {
   pub width: usize,
   /// Height of the frames in pixels.
   pub height: usize,
+  /// Display aspect ratio (for anamorphic video).
+  pub aspect_ratio: Rational,
```

Comment:
This needs to specify if it's a SAR or DAR.

Reply:
Clarified it.